org.regular_dev.biplanes_revival.yml

```
@@ -0,0 +1,34 @@
+id: org.regular_dev.biplanes_revival
+runtime: org.freedesktop.Platform
+runtime-version: '23.08'
+sdk: org.freedesktop.Sdk
+command: BiplanesRevival
+finish-args:
+  - --share=ipc
+  - --share=network
+  - --socket=wayland
+  - --socket=fallback-x11
+  - --socket=pulseaudio
+  - --device=dri
+  - --env=BIPLANES_ASSETS_ROOT=/app/share/biplanes-revival/
+modules:
+  - name: BiplanesRevival
+    buildsystem: cmake-ninja
+    config-opts:
+      - -DCMAKE_BUILD_TYPE=Release
+    build-commands:
+      - ninja
```

```suggestion
    post-install:
```

You use build system `cmake-ninja`. So why do you call `ninja`

---

I thought it was necessary to call ninja explicitly